infrastructure-playbooks/cephadm-adopt.yml

```diff
@@ -233,26 +233,38 @@
       run_once: true
       delegate_to: '{{ groups[mon_group_name][0] }}'
 
-    - name: generate cephadm ssh key
-      command: "{{ ceph_cmd }} cephadm generate-key"
-      changed_when: false
-      run_once: true
-      delegate_to: '{{ groups[mon_group_name][0] }}'
+    - name: configure root ssh key for remote connections
+      when: cephadm_ssh_user == 'root'
```

Comment:
Not sure this is the right way, I'm thinking about scenarios where you have a root user (not in the OSP context) but you don't have to generate ssh keys because they're already there.
This condition is not blocking me, but it may be something to consider (and improve) for standalone Ceph clusters.